pkgs/tools/archivers/unrar/default.nix

```diff
@@ -17,21 +17,29 @@ stdenv.mkDerivation rec {
   '';
 
   buildPhase = ''
+    # `make {unrar,lib}` call `make clean` implicitly
+    # move build results to another dir to avoid deleting them
+    mkdir -p bin
+
     make unrar
-    make clean
+    mv unrar bin
+
     make lib
+    mv libunrar.so bin
+
+    make clean
```

Comment:
I don't think a final `make clean` is necessary.

Reply:
It was there before and I think it is used to avoid copying unnecessary things to the `dev` output.

Reply:
But according to the diff it wasn't there at the end, that's the reason of my comment. The final line was previously `make lib`.

The `make clean` was there before `make lib`, and by the comment now `make lib` calls `clean` implicitly, that's why it's removed.

Reply:
Ah, understood what you meant.

But still, I think we need it to avoid copying unnecessary things at the output. At most it doesn't hurt.

Reply:
Just removed it.

Reply:
Output trees look good to me, and only headers in `$dev`.